cvat/apps/engine/views.py

```diff
@@ -242,24 +257,77 @@ def annotations(self, request, pk):
                 except (AttributeError, IntegrityError) as e:
                     return Response(data=str(e), status=status.HTTP_400_BAD_REQUEST)
                 return Response(data)
+        elif request.method == 'POST':
+            queue = django_rq.get_queue("default")
```

Comment:
Code for Task and for Job looks like very similar. Maybe it would be better, to put it into a different function